src/components/layouts/SecondMenu/SecondMenu.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import get from 'lodash/get'
 import cn from 'classnames'
 import { connect } from 'react-redux'
 import { Image, Link, Icon } from 'src/components/common'
 import { userSelector } from 'src/store/user/selectors'
+import PersonModel from 'src/api/backend/model/PersonModel'
+import { personProfile, personProfileLoadingStateSelector, personProfileSelector } from 'src/store/secondMenu'
 import { schemaFactory as accountTypesSchemaFactory } from 'src/models/app/UserAccountTypesModel'
 import css from './SecondMenu.scss'
 
 class SecondMenu extends React.Component {
   static propTypes = {
     accountTypes: PropTypes.shape(accountTypesSchemaFactory()),
+    personProfile: PropTypes.instanceOf(PersonModel),
+    loadPersonProfile: PropTypes.func,
+  }
+
+  componentDidMount () {
+    this.props.loadPersonProfile()
+  }
```

Comment:
you dont need to fetch data in this component
checkout src/store/auth/signin/actions#submit
when user logins we makeing signin in profile service which retunrs user profile
then we put this profile into store in store.user.user.priofle by userSave action
so you just need connect already stored profile to this component
please refactor this and remove store for second menu component